### PR TITLE
Fix presubmit parsing issue with emails.

### PR DESCRIPTION
**Pull request by inferno-chromium**

### CODE REVIEW

**Comment by inferno-chromium** [comment]
TBR to unblock devs.

File: infra/presubmit.py [hunk 1]
```diff
@@ -197,6 +201,7 @@ def do_checks(changed_files):
     '.sh',
     '.yaml',
 ]
+_CHECK_LICENSE_EXCLUSIONS = ['project.yaml']
```
Comment (Dor1s) [inline infra/presubmit.py]: should we delete line 202 to skip license checks for all .yaml files?

Comment (inferno-chromium) [reply infra/presubmit.py]: Upto you, i am fine with either ? Which other yaml files you can think of ? Ideally all of them should :(

Comment (Dor1s) [reply infra/presubmit.py]: I can't think of any .yaml files in OSS-Fuzz repo that need license check

Comment (inferno-chromium) [reply infra/presubmit.py]: Ok feel free to remove that ?

**Comment by jonathanmetzman** [comment]
What was the issue previously? That primary_contact field is a string not a list of strings?

**Pull request merged**